src/crowdsale/CrowdSale.sol

```diff
@@ -194,7 +203,7 @@ contract CrowdSale {
      * @param saleId the sale id
      * @param permission. bytes are handed over to a configured permissioner contract
      */
-    function claim(uint256 saleId, bytes memory permission) public returns (uint256 auctionTokens, uint256 refunds) {
+    function claim(uint256 saleId, bytes memory permission) public whenNotPaused returns (uint256 auctionTokens, uint256 refunds) {
```

Comment:
feels weird to pause the claim/settle functionality. this basically gives the owner the power to lock in funds